### PR TITLE
Feat/game library

### PR DESCRIPTION
**Pull request by u22520016**
I changed some stuff so that the GitHub Actions pass the checks.

### CODE REVIEW

File: lib/theme/themes.dart
Comment (u22520016) [inline lib/theme/themes.dart:1]: I changed the different colours to be used in the app.  The font needs to be added again.  I also changed the way the light and dark mode works (but it should be automatic when the theme is changed).

**Comment by u22520016** [comment]
I added a comment to the themes file that explain what the changes are.

File: lib/theme/themes.dart
Comment (MonikaTheiss) [reply lib/theme/themes.dart:1]: thank you

**Review by MonikaTheiss — APPROVED** [review]
The game library looks super cool !!  Thank you 😄 🥳

**Comment by pieryvdl** [comment]
I see that the Cached Network Image was removed from the code. Can I add it back to the correct place to make the loading a bit faster?

**Comment by u22520016** [comment]
> I see that the Cached Network Image was removed from the code. Can I add it back to the correct place to make the loading a bit faster? The new page looks great though. Thanks Franco!

Yes you can

**Review by pieryvdl — COMMENTED** [review]
Just want to add the Cached Network Image before we commit this, but it looks great.

**Review by pieryvdl — APPROVED** [review]
Images load faster, we can merge this in.

**Review by MonikaTheiss — APPROVED** [review]

**Pull request closed without merging**